model/enduser/common.yaml

```diff
@@ -0,0 +1,22 @@
+groups:
+  - id: identity
+    type: attribute_group
+    brief: >
+      These attributes may be used for any operation with an authenticated and/or authorized enduser.
```

Comment:
```suggestion
      Describes end user identity. 
```

Reply:
yes, `enduse.pseudo.id` can be unauthenticated, unauthorized, anonymous, and etc.. as long as it's not authenticated, which is tracked under `enduser.authentication.id`.